docs/governance/process.md

```diff
@@ -18,10 +18,7 @@ In the past, different people have considered contributions amounts differently.
 
 #### Burned deposits
 
-Deposits are burned when proposals:
-1. **Expire** - deposits will be burned if the deposit period ends before reaching the minimum deposit (64 ATOM)
-2. **Fail** to reach quorum - deposits will be burned for proposals that do not reach quorum ie. 40% of all staked ATOM must vote
-3. **Are vetoed** - deposits for proposals with 33.4% of voting power backing the 'NoWithVeto' option are also burned
+Deposits are burned only when proposals are vetoed i.e. 33.4% of voting power backing the 'NoWithVeto' option.
```

Comment:
Is it worth adding a link to https://github.com/cosmos/cosmos-sdk/blob/main/x/gov/spec/01_concepts.md#deposit-refund-and-burn ?

Reply:
makes sense. see below